docs/sdk_developers/rebasing.md

```
@@ -1,121 +1,64 @@
-git commit -s -S -m "Your-commit-message"
-```
-
-### 2. Rebase your branch on top of the updated main
-# Rebase = cleaner history, your commits appear on top of main.
+### 3. Verify Sign Status
+Verify after the rebase operation, your n commits are still signed correctly:
 
 ```bash
-git rebase main -S
+git log -n --pretty=format:'%h %an %G? %s'
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)